environments/skeleton/{{cookiecutter.environment}}/inventory/group_vars/all/cloud_init.yml

```diff
@@ -3,17 +3,19 @@
 # Note volumes MUST be defined using block_device{} inside openstack_compute_instance_v2,
 # NOT attached after boot, else the device ordering is liable to change e.g. on reboots.
 
+block_device_prefix: sd
+
 cloud_init_userdata_templates_extra:
   - module: fs_setup
     group: control
     template: |
         - label: state
           filesystem: ext4
-          device: /dev/vdb
+          device: /dev/{{ block_device_prefix }}b
```

Comment:
Don't mind how this is done, but it would be quite nice to have it the same on both sides: https://github.com/stackhpc/caas-slurm-appliance/blob/726ba77e7870b1586dc078d048c347499a32a710/group_vars/openstack.yml#L23-L24